frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsLoader.tsx

```diff
@@ -23,7 +24,8 @@ const QuickStartsLoader: React.FC<QuickStartsLoaderProps> = ({ children }) => {
       permissionChecks.current[quickStart.metadata.name] = hasPermission;
       if (Object.keys(permissionChecks.current).length === quickStarts.length) {
         const filteredQuickStarts = quickStarts.filter(
-          (quickstart) => permissionChecks.current[quickstart.metadata.name],
+          (quickstart) =>
+            permissionChecks.current[quickstart.metadata.name] && !isDisabledQuickStart(quickstart),
```

Comment:
Filter out the disabled quick start from the original list which is returned by useK8sWatchResources that way we can avoid permission checks for the disabled quick starts

Reply:
updated